### PR TITLE
e2e/authorization: simplify

### PR DESCRIPTION
**Pull request by sttts**
Use resource bootstrapper and simplify test.

### CODE REVIEW

File: test/e2e/authorizer/authorizer_test.go [hunk 1]
```diff
@@ -151,32 +104,43 @@ func TestAuthorizer(t *testing.T) {
...
-			Dynamic:    dynamicClient,
+		"org": {
+			KubeClient: orgKubeClient,
+			KcpClient:  orgKcpClient,
+			Dynamic:    orgDynamicClient,
 		},
-		"user-1": newUserClient(t, "user-1", "admin", "workspace1", kcpCfg),
-		"user-2": newUserClient(t, "user-2", "admin", "workspace1", kcpCfg),
-		"user-3": newUserClient(t, "user-3", "admin", "workspace1", kcpCfg),
+		"user-1": newUserClient(t, "user-1", helper.EncodeOrganizationAndWorkspace(org, "workspace1"), kcpCfg),
```
Comment (marun) [inline test/e2e/authorizer/authorizer_test.go:131]: nit: Maybe defer this call to newUserClient to simplify the call?

File: test/e2e/authorizer/authorizer_test.go [hunk 2]
```diff
@@ -151,32 +104,43 @@ func TestAuthorizer(t *testing.T) {
...
 	require.NoError(t, err)
-	kcpClients, err := kcp.NewClusterForConfig(kcpCfg)
+	dynamicClusterClient, err := dynamic.NewClusterForConfig(kcpCfg)
 	require.NoError(t, err)
-	kcpClient := kcpClients.Cluster(clusterName)
-	require.NoError(t, err)
-	dynamicClient, err := dynamic.NewForConfig(kcpCfg)
+
+	orgClusterName := "admin"
+	_, org, err := helper.ParseLogicalClusterName(orgClusterName)
```
Comment (marun) [inline test/e2e/authorizer/authorizer_test.go:118]: What does this function call accomplish?

Comment (s-urbaniak) [reply test/e2e/authorizer/authorizer_test.go:118]: from what i see just validation

**Review by s-urbaniak — APPROVED** [review]

File: test/e2e/authorizer/authorizer_test.go (hunk 2 above)
Comment (sttts) [reply test/e2e/authorizer/authorizer_test.go:118]: for now it does nothing. When orgs become a thing in #508, this will make more sense.

**Pull request merged**